Software/dist_linux/flatpak/manifest.yml.template

```diff
@@ -9,7 +9,7 @@ finish-args:
 
   - --share=network
   - --socket=pulseaudio
-  - --filesystem=~/.Prismatik:rw
+  #- --filesystem=~/.Prismatik:rw
```

Comment:
Please add a comment explaining why, or remove the line entirely

Reply:
I commented this out as explained in point 3 of the original comment. I will remove this line entirely.

Reply:
Thanks!